content/blog/2020-05-08-dvc-ambassador-program.md.md

```diff
@@ -0,0 +1,151 @@
+  in
+
+The program is structured to provide a lot of flexibility, so each ambassador
+can do outreach in ways that are personally motivating and enjoyable. There are
+a few guidelines:
+
+- We ask for a one-year commitment
+- We ask ambassadors to contribute four activities per year, about once every
+  three months.
+- For your first contribution, we ask for a blog post- this way, we can
+  collaborate with you to help get all the technical details right. After that,
+  it’s up to you!
+
+## Some ideas to get started
+
+Our official ambassador program is just starting, but our community already has
+a lot of folks making noise. Here are just a few contributions we admire- we
+think they’re pretty cool inspirations for future projects.
+
+### Blogs and tutorials
+
+Shareable blogs are one of our most effective outreach strategies. They give
+visibility to the author _and_ new ways to use DVC, so it's a win-win.
+
+- [Remote training with GitLab-CI and DVC](https://blog.codecentric.de/en/2020/01/remote-training-gitlab-ci-dvc/),
+  by Mercel Mikl and Bert Besser (Bert has also organized a DVC meetup in
+  Berlin)
+- [Creating a solid Data Science development environment](https://towardsdatascience.com/creating-a-solid-data-science-development-environment-60df14ce3a34),
+  by Gabriel dos Santos Goncalves
+- [Continuous Delivery for Machine Learning](https://martinfowler.com/articles/cd4ml.html),
+  by Danilo Sato, Arif Wider, and Christoph Windheuser
+- [Manage your Data Science Project in R](https://mribeirodantas.xyz/blog/index.php/2020/03/05/r-dvc-and-rmarkdown/)
+  was my first blog post about using DVC in an R project!
+
+### Talks
+
+Community members have presented at events like PyCon, PyData, and local
+meetups.
+
+- [Version control for data science](https://www.slideshare.net/AlessiaMarcolini/version-control-for-data-science),
+  by Alessia Marcolini @ PyCon DE & PyData Berlin
+- [How to easily set up and version control your machine learning pipelines](https://www.youtube.com/watch?v=rUTlqpcmiQw),
+  by Sarah Diot-Girard & Stephanie Bracaloni @ PyData Amsterdam
+- [ML models and dataset versioning](https://speakerdeck.com/kurianbenoy/ml-models-and-dataset-versioning),
+  by Kurian Benoy @ PyCon India
+
+### Code contributions
+
+Our GitHub repository has lots of open discussions about potential features- its
+a goldmine for ways to pitch in. For example:
+
+- [Helge Munk Jacobsen](https://github.com/elgehelge) took on an open issue in
+  our code base about supporting hyperparameter tracking with DVC and made a
+  pull request to add this feature.
+
+- [Vera Sativa](https://github.com/verasativa/) added directory support to the
+  `dvc import-url` function- and she was our 100th contributor, so she won her
+  own
+  [DeeVee the Owl](https://dvc.org/static/78b685e283d679c8ebe518ea17520f6d/75999/odd_with_deevee.png).
+
+If any of this sounds fun to you, please be in touch. We look forward to
```

Comment:
You can provide contacts one more time. Email. Twitter might also help.

Reply:
Good point!